source/atlas-cli-deploy-local.txt

```diff
@@ -86,6 +86,105 @@ Before you begin, complete the following prerequisites:
 
 .. include:: /includes/steps-atlas-cli-deploy-local-prereqs.rst
 
+.. _atlas-cli-pvt-registry-setup:
+
+(Optional) Set up a Private Image Registry 
+------------------------------------------
+
+This section demonstrates how to pull and tag images, load images to a
+container registry, and configure the {+atlas-cli+} to use private
+images for building local clusters. 
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Install the Docker command line tools.
+   
+   .. step:: Create your container registry with the namespace and name of repository. 
+
+   .. step:: Fetch the MongoDB image for local deployment. 
+
+      To fetch, run the following command after replacing the following
+      placeholder values:
+
+      .. list-table:: 
+         :stub-columns: 1 
+
+         * - ``<version-number>``
+           - The version number of {+atlas-cli+} that you want to use in
+             your deployment. 
+
+         * - ``<container-registry-namespace>`` 
+           - The fully qualified path and name of the container registry
+             namespace that you created in step 2.
+    
+      .. code-block:: shell 
+
+         docker tag mongodb/mongodb-atlas-local:<version-number> <container-registry-namespace>/mongodb-atlas-local:<version-number>
+         
+   .. step:: Push the image on your machine to your container registry.
+
+      a. Run the following command after replacing the following
+         placeholder values to push the image to your container
+         registry: 
+
+         .. list-table:: 
+            :stub-columns: 1 
+
+            * - ``<version-number>``
+              - The version number of {+atlas-cli+} that you want to use in
+                your deployment. 
+
+            * - ``<container-registry-namespace>`` 
+              - The fully qualified path and name of the container registry
+                namespace that you created in step 2.
+
+         .. code-block:: shell
+
+            docker push <container-registry-namespace>/mongodb-atlas-local:<version-number>
+
+         It might take a few minutes for this operation to complete.
+
+      #. Login to your container registry and verify that the image was
+         successfully uploaded. 
```

Comment:
It might be nice to add a validation recommendation here for the user to try pulling the image: "... successfully uploaded, and that you can pull it to your machine using `docker pull`".